src/colors/conversions.jl

```diff
@@ -0,0 +1,45 @@
+"""
+    channelify(m::AbstractArray{<:Color})
+Channelify function.
+
+Input should be in ``WHN`` order ``\\implies (*, batch)``.
+Output will be in the order ``(*, channels, batch)``.
+
+"""
+function channelify(m::AbstractArray{T}) where T <: Color
+    m = channelview(m)
+    t = ntuple(i->i, ndims(m))
+    if T <: AbstractGray
+        m = unsqueeze(m, t[end-1])
+    end
```

Comment:
For the record, this seemly inconsistent behavior for `Gray` is decided in https://github.com/JuliaImages/ImageCore.jl/issues/97 so it's not going to change.

Reply:
This has been specially done because Flux required the dimensions to be in `(height, width, channels, batch)` format, so it would be inconsistent with Flux.